### PR TITLE
Yaels code

### PR DESCRIPTION
**Pull request by YaelBorn**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,9 +16,25 @@
...
 <body>
-
-  <!-- START OF BUILD -->
-   
+  <main>
+    <div class="card">
+      <img src="./images/illustration-hero.svg"> 
+    <h1> Order Summary </h1>
+    <p>You can now listen to millions of songs, audiobooks, and podcasts on any device anywhere you like!</p>
+    <div class="card2">
```
Comment (opusxj) [inline index.html]: card2 is not a very good class name, can we change this.

File: index.html [hunk 2]
```diff
@@ -16,9 +16,25 @@
   <title>Frontend Mentor | Order summary card</title>
 </head>
 <body>
-
-  <!-- START OF BUILD -->
-   
+  <main>
+    <div class="card">
+      <img src="./images/illustration-hero.svg">
```
Comment (opusxj) [inline index.html]: Where is the alt tag?

File: style.css [hunk 3]
```diff
@@ -1,5 +1,78 @@
...
+	min-height: 100vh;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.card {
+	max-width: 495px;
+	width: 100%;
+	background-color: white;
```
Comment (opusxj) [inline style.css]: Can we change this to be a hex value please.

File: style.css [hunk 4]
```diff
@@ -1,5 +1,78 @@
...
+}
+
+p {
+	text-align: center;	
+	font-weight: 500;
+	font-size: 16px;
+	color: #717FA6;
+	margin-top: 17px;
+	max-width: 354px;
+	margin-left: auto;
```
Comment (opusxj) [inline style.css]: can we shorthand this margin.

File: style.css [hunk 5]
```diff
@@ -1,5 +1,78 @@
...
+	margin-right: auto;
+	line-height: 26px;
+}
+
+.card2 {
+	background-color: #F7F9FF;
+	max-width: 354px;
+	width: 100%;
+	margin-left: auto;
+	margin-right: auto;
```
Comment (opusxj) [inline style.css]: same as above

File: style.css [hunk 6]
```diff
@@ -1,5 +1,78 @@
...
+	font-weight: 900;
+	font-size: 15px;
+	margin-top: 32px;
+	margin-left: auto;
+	margin-right: auto;
+	text-align: center;	
+	max-width: 354px;
+	width: 100%;
+	border-radius: 11px;
+	box-shadow: 0 20px 20px 0px rgb(56 42 225 / 19.03%);
```
Comment (opusxj) [inline style.css:60]: Lovely <3

File: style.css [hunk 7]
```diff
@@ -1,5 +1,78 @@
...
+	padding: 15px 102px;
+}
+
+.cancel {
+	color: #717FA6;
+	font-weight: 900;
+	font-size: 15px;
+	margin-top: 32px;
+	margin-left: auto;
```
Comment (opusxj) [inline style.css]: Can we shorthand this margin as well

File: style.css [hunk 8]
```diff
@@ -1,5 +1,78 @@
...
+	border-radius: 11px;
+}
+
+.button {
+	background-color: #382AE1;
+	color: #FFFFFF;
+	font-weight: 900;
+	font-size: 15px;
+	margin-top: 32px;
+	margin-left: auto;
```
Comment (opusxj) [inline style.css]: And again

File: index.html [hunk 9]
```diff
@@ -16,9 +16,25 @@
...
+    <div class="card">
+      <img src="./images/illustration-hero.svg"> 
+    <h1> Order Summary </h1>
+    <p>You can now listen to millions of songs, audiobooks, and podcasts on any device anywhere you like!</p>
+    <div class="card2"> 
+      Annual plan
+    </div>
+
+    <div class="button"> 
+      Proceed to payment
```
Comment (opusxj) [inline index.html]: Why is this not a button?

File: index.html [hunk 10]
```diff
@@ -16,9 +16,25 @@
...
+    </div>
+
+    <div class="cancel">
```
Comment (opusxj) [inline index.html]: Same again, can we add a button

**Review by opusxj — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (YaelBorn) [reply index.html]: turned card2 into plan

File: index.html (hunk 2 above)
Comment (YaelBorn) [reply index.html]: added and alt tag

File: style.css (hunk 3 above)
Comment (YaelBorn) [reply style.css]: yes sir

File: style.css (hunk 4 above)
Comment (YaelBorn) [reply style.css]: Done :)

File: style.css (hunk 5 above)
Comment (YaelBorn) [reply style.css]: Done <3

File: style.css (hunk 6 above)
Comment (YaelBorn) [reply style.css:60]: thank you sensei

File: style.css (hunk 7 above)
Comment (YaelBorn) [reply style.css]: yes sir

File: style.css (hunk 8 above)
Comment (YaelBorn) [reply style.css]: done :)

**Pull request merged**